services/src/test/java/org/exoplatform/onlyoffice/OnlyofficeEditorServiceTest.java

```diff
@@ -354,31 +553,248 @@ public void testSetLastModifier() throws Exception {
   }
 
   /**
-   * Test update title for not root node
+   * Test update Document fot status code 3 it's an error of saving in
+   * Onlyoffice, we sync to remote editors list first
    */
   @Test
-  public void testUpdateTitle() throws Exception {
+  public void testUpdateDocumentErrorSavingInOnlyOffice() throws Exception {
     // Given
     startSessionAs("john");
-    Node node = createDocument("Test Document.docx", "nt:file", "testContent", false);
+    Node node = createDocument("Test Document.docx", "nt:file", "testContent", true);
+    Config config = editorService.createEditor("http", "127.0.0.1", 8080, "john", null, node.getUUID());
+    DocumentStatus status = new DocumentStatus.Builder().status(3L)
+                                                        .users(new String[] { "john" })
+                                                        .userId("john")
+                                                        .url("http://127.0.0.1:8080/editor/")
+                                                        .key(config.getDocument().getKey())
+                                                        .build();
 
     // When
-    editorService.updateTitle("portal-test", node.getUUID(), "TestDocumentJohn.docx", "john");
+    editorService.updateDocument(status);
 
     // Then
-    assertFalse(node.getParent().hasNode("Test Document.docx"));
-    assertTrue(node.getParent().hasNode("TestDocumentJohn.docx"));
+    assertNotNull(status.getConfig());
+    assertNotNull(status.getConfig().getError());
     node.remove();
   }
 
   /**
-   * Test update title for root node ("/")
+   * Test update Document for unknown status code
+   * PAS D'ASSERT lOG.warm
    */
   @Test
-  public void testUpdateTitleRootNode() throws Exception {
+  public void testUpdateDocumentForUnknownCode() throws Exception {
+    // Given
+    startSessionAs("john");
+    Node node = createDocument("Test Document.docx", "nt:file", "testContent", true);
+    Config config = editorService.createEditor("http", "127.0.0.1", 8080, "john", null, node.getUUID());
+    DocumentStatus status = new DocumentStatus.Builder().status(5L)
+                                                        .users(new String[] { "john" })
+                                                        .userId("john")
+                                                        .saved(true)
+                                                        .key(config.getDocument().getKey())
+                                                        .build();
+
+    // When
+    editorService.updateDocument(status);
```

Comment:
Can we use editorService.getDocumentById to retrieve the document and make some assertions on it ?